pkg/api/server.go

```diff
@@ -0,0 +1,43 @@
+package api
+
+import (
+	"context"
+
+	"github.com/inngest/inngest-cli/pkg/server"
+)
+
+func NewServer() server.Server {
+	return &apiServer{}
+}
+
+type apiServer struct {
+	api *API
+}
+
+func (a *apiServer) Name() string {
+	return "api"
+}
+
+func (a *apiServer) Pre(ctx context.Context) error {
+	var err error
+
+	// TODO: Conenct and load ingest keys.
+	a.api, err = NewAPI(Options{
+		Hostname: "localhost",
```

Comment:
This is only a demo prior to adding configuration via Cue;  I'll start work on the config package now